Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3
+
+WORKDIR /procrastinate_dev/
+COPY requirements.txt ./
+COPY setup.* ./
+COPY procrastinate ./procrastinate/
```

Comment:
You can have just one COPY instruction I guess:

```
COPY requirements.txt setup.* procrastinate ./
```

no?

(I am not sure about the recursive copy of the procrastinate directory.)

Reply:
I'll try that, thanks!

Reply:
I cannot copy the `procrastinate/` directory on the same line, but I merged the first two copies.